deployment/config.go

```diff
@@ -179,6 +183,44 @@ type ExternalBucketSettings struct {
 	AmazonS3SSE             bool   `default:"false"`
 }
 
+// ExternalAuthProviderSettings contains the necessary data
+// to configure an external auth provider.
+type ExternalAuthProviderSettings struct {
+	// Enabled is set to true if the external auth provider should be enabled.
+	Enabled bool `default:"false"`
+	// DevelopmentMode is set to true if the keycloak instance should be started in development mode.
+	DevelopmentMode bool `default:"true"`
```

Comment:
Should it not be "false" by default? The default config is always aimed towards production usage.

Reply:
I left it as `false` because the difference development vs production means. If I use the production option I have to disable some of the features it enables manually (like enforced hostnames, SSL, local cache). I could go that route but in the end we would have the same result but with more work. That is, unless we want to really run it in production mode with all security features enforced as they should be.